_apps/bowtie.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Bowtie - A short read aligner
```

Comment:
```suggestion
title: <b>Bowtie</b> - A short read aligner
```

Can you do this for the others, too?
Sometimes names have a `-` in them and I think it's easier to parse when the main name is bold (needs to be html tag, md doesn't work here)